StripeConnect/StripeConnect/Source/Components/AccountManagementViewController.swift

```diff
@@ -22,39 +22,31 @@ public class AccountManagementViewController: UIViewController {
         }
     }
 
-    let webView: ConnectComponentWebView
+    private(set) var webVC: ConnectComponentWebViewController!
 
     public weak var delegate: AccountManagementViewControllerDelegate?
 
     init(componentManager: EmbeddedComponentManager,
-         collectionOptions: AccountCollectionOptions) {
-        webView = ConnectComponentWebView(
+         collectionOptions: AccountCollectionOptions,
+         loadContent: Bool) {
+        super.init(nibName: nil, bundle: nil)
+        webVC = ConnectComponentWebViewController(
             componentManager: componentManager,
-            componentType: .accountManagement
+            componentType: .accountManagement,
+            loadContent: loadContent
         ) {
             Props(collectionOptions: collectionOptions)
-        }
-        super.init(nibName: nil, bundle: nil)
-
-        webView.addMessageHandler(OnLoadErrorMessageHandler { [weak self] value in
+        } didFailLoadWithError: { [weak self] error in
```

Comment:
Made the `didFailLoadWithError` a required property passed on `init`. This now gets used in more than just the `OnLoadErrorMessageHandler` but also when the web page fails to load.

By making it an argument passed on init, it ensures we never forget to set the handler. It does force us to make `webVC` an implicitly unwrapped optional so that we can reference `self` when initializing the webVC, but personally I think it's worth the tradeoff and better than forgetting to set the `didFail` handler.